### DISCUSSION TITLE
Running remote Twister tests against main for baseline comparison

### DISCUSSION THREAD

**Discussion by snematbakhsh** [zephyrproject-rtos/zephyr#42331] [Q&A]
Hello Developer / Testing Community,

Several on-board tests are failing on my recent pull request (https://github.com/zephyrproject-rtos/zephyr/runs/4987549151?check_suite_focus=true) for a board that I do not have and is no longer in production. Is there a simple way that I can trigger specific tests for specific board(s) be run on main for comparison purpose? I think I could create a "fake" pull request that touches the meaningful files to trigger my wanted tests. But maybe there is a better way?

Follow-up question: Is there a way I can search the results for the most recent N runs of a given test on a given board? I found https://github.com/zephyrproject-rtos/zephyr/actions/workflows/twister_tests.yml, but the search options don't seem sufficient.

**Comment by nashif** [comment]
You can also run twister locally on the boards you want to test, you shall not depend on CI only.
